client/src/components/Tickets/TicketStandaloneCreate.test.js

```diff
@@ -3,7 +3,8 @@ import configureStore from 'redux-mock-store';
 import '@testing-library/jest-dom/extend-expect';
 import renderComponent from 'utils/testing';
 import TicketStandaloneCreate from './TicketStandaloneCreate';
-import { initialTicketCreateState as ticketCreate } from '../../redux/reducers/tickets.reducers';
+import { initialTicketCreateState as ticketCreate} from '../../redux/reducers/tickets.reducers';
+import {initialState as workbench} from '../../redux/reducers/workbench.reducers';
```

Comment:
```suggestion
import { initialTicketCreateState as ticketCreate } from '../../redux/reducers/tickets.reducers';
import { initialState as workbench } from '../../redux/reducers/workbench.reducers';
```
Nit-picky style stuff.